changelog.md

```diff
@@ -1,5 +1,65 @@
 # Changelog
 
+## 1st April 2021
```

Comment:
I'd put there "Planned changes for 1st April 2021" so that it doesn't look like we're time traveling. We're hospitality system of the future though :D :D Anyway, I like it this way 👏